src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age ? <p className="Person__age">I am {person.age}</p> : ''}
+    {person.sex === 'm' && person.isMarried === true ? (
+      <p className="Person__patner">{person.partnerName} is my wife</p>
+    ) : (
+      <p className="Person__patner">{person.partnerName} is my husband</p>
+    )}
+    {person.isMarried === false ? (
+      <p className="Person__patner">I am not married</p>
+    ) : (
+      ''
+    )}
```

Comment:
decide before jsx what text will be rendered and just render in jsx. something like that

```
const personPartnerType = sex === 'm' ? 'wife' : 'husband';
const personPartner = isMarried ? `${partnerName} is my ${personPartnerType}` : 'I am not married'
```